knapsack_rspec_report.json

```
@@ -27,7 +27,6 @@
   "spec/presenters/setup_presenter_spec.rb": 0.05804999999963911,
   "spec/features/remember_device/session_expiration_spec.rb": 5.3716609999974025,
   "spec/requests/redirects_spec.rb": 0.12476699999388075,
-  "spec/lib/feature_management_spec.rb": 158.021550999998,
```

I couldn't figure out how to regenerate the weight for this... so figured it was just easier to remove 😬

---

`KNAPSACK_GENERATE_REPORT=true rspec` is the way I've done in the past, but that's for all tests, and it takes a very long time to run.

You could generate it for the one file and merge the resulting report into the original file, but I'm not sure if that weight would be accurate on its own without anything to compare it to (unsure if the number is an absolute weight or a relative weight, and unsure if it varies from computer to computer).

```
KNAPSACK_GENERATE_REPORT=true rspec spec/lib/feature_management_spec.rb
```

This is what I got when doing that:

```suggestion
  "spec/lib/feature_management_spec.rb": 0.27332799999567214,
```

---

Maybe I typo'd the env variable, but it didn't write that file for me. Anyways this is great, thanks!

And I'm almost certain the values in here are a number of seconds, so they're essentially absolute weights

---

updated in 3a0712bfa2d573d07fa2eaaeb71ebd677916f1ff

---

> And I'm almost certain the values in here are a number of seconds

Yep, it's seconds